src/views/ResourceManagement/CapacityOnDemand/CapacityOnDemandActivation.vue

```diff
@@ -46,7 +46,7 @@
                   </b-form-invalid-feedback>
                 </b-input-group>
               </b-form-group>
-              <b-col align-self="baseline">
+              <b-col sm="2" align-self="baseline">
```

Comment:
The issue still occurs with different screen sizes.
![image](https://user-images.githubusercontent.com/96164786/205019837-8e21ed68-7015-430e-8d2a-ab1c34b0bf0b.png)

Reply:
Thanks @Nikhil-Ashoka 
So, could you close the status bar on the right and check again?

Reply:
Yes, It works fine for the full screen in my system. However, If we are using a smaller screen, This issue is recurring.

Reply:
What size screen did you use to test? Let's see if this problem is expected.

Reply:
Eg: Responsive: 770*1280 to 830*1280. This issue has occurred again.